script.js

```diff
@@ -67,7 +67,7 @@ export async function script(octokit, repository, options) {
 
       renovateConfigObj.extends = newExtends;
 
-      return JSON.stringify(jsonFile);
+      return JSON.stringify(jsonFile, null, "\t");
```

Comment:
we don't want to use `\t`, let's use two spaces `  ` plus a `\n` at the end. @oscard0m I'll open a PR